index.html

```diff
@@ -0,0 +1,321 @@
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/medium.png" alt="medium-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/github.png" alt="github-icon"
+              /></a>
+            </li>
+            <li>
+              <a href="#"
+                ><img src="./images/angellist.png" alt="angellist-icon"
+              /></a>
+            </li>
+          </ul>
+          <img src="./images/resume.png" alt="button" class="press">
+        </div>
+        <div class="lang">
+          <ul class="start"> 
+            <li class="lan">Languages</li>
+            <li class="material-symbols-outlined">expand_more</li>
+          </ul>
+          <ul class="lang-icons">
+            <li>
+              <img src="./images/Frame-9.png" alt="js">
+            </li>
+            <li>
+              <img src="./images/Frame-10.png" alt="html">
+            </li>
+            <li>
+              <img src="./images/Frame-11.png" alt="css">
+            </li>
+          </ul>
+          <div class="end">
+            <ul class="frame">
+              <li>Frameworks</li>
+              <li class="material-symbols-outlined">navigate_next</li>
+            </ul>
+            <hr />
+            <ul class="skills">
+              <li>Skills</li>
+              <li class="material-symbols-outlined">navigate_next</li>
+            </ul>
+            <hr />
+          </div>
+        </div>
+      </div>
+    </section>
+    <footer class="contact-part">
+      <h2 class="header">Contact me</h2>
+      <div class="bodie">
+        <div class="text">
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <form action="https://formspree.io/f/mjvlpwbr" method="post">
+          <ul class="formie">
+            <li>
+              <input
+                type="text"
+                name="name"
+                id="name"
+                maxlength="30"
+                class="field"
+                placeholder="Thomas"
+                required
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                name="example@gmail.com"
+                placeholder="example@gmail.com"
+                class="mail"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                name="message"
+                maxlength="500"
+                placeholder="Write your message here"
+                class="typing"
+                rows="10"
+                required
+              ></textarea>
+            </li>
+          </ul>
+          <img src="./images/in-touch.png" alt="button" class="in-touch">
+        </form>
+      </div>
```

Comment:
- [ ] You have used an image in place of a `<button>` tag for the "Get in touch" button. Please remove the image and use a button tag instead. Do this for all your buttons.
- [ ] Also kindly implement the button default, hover, and active states as shown in the Figma design in my earlier suggestions ☝️